server/controllers/logout.js

```diff
@@ -0,0 +1,18 @@
+
+const handleLogout = async(req, res) => {
+  const token = req.cookies.jwt || '';
+  if (!token) {
+    return res.status(401).json({ message: 'You need to Login' });
+  }
+  else {
+    //return res.status(202).clearCookie('jwt').json({ message: 'Logged out' });
+    await res.clearCookie('jwt');
```

Comment:
i don't think you need to await for this, it should happen immediately

Reply:
Removed the async and await as no synchronous call inside the function